### DISCUSSION TITLE
Increase the Image number from the default 32

### DISCUSSION THREAD

**Discussion by EmeraldGirl** [lllyasviel/Fooocus#2868] [Q&A] [answered]
Is it possible to increase the image generated quantity above 32 please ?

**Comment by ILikeToasters** [comment, marked as answer]
Yes. One option is if you right click on the generate button it will give the option to generate forever. The other option is to edit your config/preset files in the Fooocus folder. There is further instructions on editing the config.txt file in your Fooocus folder in the config_modification_tutorial.txt file. 
<img width="371" alt="Screenshot 2024-05-04 161346" src="https://github.com/lllyasviel/Fooocus/assets/4357657/c03361e1-ab3a-41c7-9920-aa8b309fa6c0">
